src/Components/CreateImageWizard/steps/Oscap/Oscap.tsx

```diff
@@ -460,6 +465,8 @@ const ComplianceSelectOption = ({ policy }: ComplianceSelectOptionPropType) => {
 // so just use the ones from the major release.
 export const removeBetaFromRelease = (dist: Distributions): Distributions => {
   switch (dist) {
+    case RHEL_10_BETA:
+      return RHEL_10;
```

Comment:
Same here, no RHEL_10 in Distributions.

Reply:
ugh right, hmm how's that going to work :confused:

Reply:
actually I think we can just return the constant and cast it?
bit hacky but should work until rhel 10 actually exists.